google/cloud/storage/internal/sha256_hash.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_STORAGE_INTERNAL_SHA256_HASH_H_
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_STORAGE_INTERNAL_SHA256_HASH_H_
+
+#include "google/cloud/storage/version.h"
```

Comment:
Why this, and why not <cstdint>?

Reply:
Ergh ... Why this and not `<cstdint>`?

Reply:
`storage/version.h` defines `STORAGE_CLIENT_NS`.  why not... were you going to propose an alternative?

Reply:
Ah, yes.  You are right of course, `cstdint` is not guaranteed to be part of the other includes. Fixed.

Reply:
Sorry about the STORAGE_CLIENT_NS thing ... I'm still trying to memorize stuff :-(.